mkdocs.yml

```diff
@@ -47,6 +47,7 @@ nav:
               - ODK - Managing imports: tutorial/managing-dynamic-imports-odk.md
               - ODK - Managing ontology releases: tutorial/managing-ontology-releases-odk.md
               - ODK - Migrating to ODK: tutorial/migrating-ontology-to-odk.md
+              - DL queries: tutorial/basic-dl-query.md
```

Comment:
Maybe better in Protege section rather than ODK?

Reply:
whoops, my bad, totally agree